profiles/sample-contacts.yaml

```diff
@@ -0,0 +1,19 @@
+contacts:
```

Comment:
This file should be renamed to <yourname>-contacts.yaml and is the file where people would put information on any characters they did not want contributed to the public file or any personal notes on characters. The method to merge the two YAMLs will give precedence to this file - any fields populated here will apply over the contents of the base file.